.gitignore

```
@@ -1 +1,3 @@
-__*__
\ No newline at end of file
+__*__
+./log/*
```

The root gitignore will generally presume a relative path structure within the context of the repo, meaning the following could work the same way with fewer keystrokes. While this is a minimal change, it can help with readability.

```suggestion
log/*
```

---

changed format of gitignore file